### PR TITLE
Dataloss in localized fields and maps

### LINKED ISSUE #6509: Dataloss in localized fields and maps
Class loses all input of localized fields and points on the maps. Using 26 languages.

[bulk_export (2).json.txt](https://github.com/pimcore/pimcore/files/4722362/bulk_export.2.json.txt)

### PR DESCRIPTION
**Pull request by weisswurstkanone**
resolves #6509

### CODE REVIEW

File: models/DataObject/Localizedfield/Dao.php [hunk 1]
```diff
@@ -167,7 +167,7 @@ public function save($params = [])
                             )
                             || (((!$container instanceof DataObject\Fieldcollection\Definition || $container instanceof DataObject\Objectbrick\Definition)
                                     && $this->model->isLanguageDirty($language))
-                                || $params['saveRelationalData']['saveLocalizedRelations'])) {
+                                || ($params['saveRelationalData']['saveLocalizedRelations'] ?? false))) {
```
Comment (blankse) [inline models/DataObject/Localizedfield/Dao.php:170]: We could simplify the if when we wrote that in a variable beforehand:
`$saveLocalizedRelations = $params['saveRelationalData']['saveLocalizedRelations'] ?? false;`

What do you think?

Comment (blankse) [reply models/DataObject/Localizedfield/Dao.php:170]: @weisswurstkanone It's not the only line. You could also save the isset calls 6/7 lines above...

File: models/DataObject/Localizedfield/Dao.php [hunk 2]
```diff
@@ -157,17 +157,14 @@ public function save($params = [])
                     }
 
                     $isUpdate = isset($params['isUpdate']) && $params['isUpdate'];
-                    $childParams = $this->getFieldDefinitionParams($fd->getName(), $language, ['isUpdate' => $isUpdate, 'context' => $context]);
+                    $childParams = $this->getFieldDefinitionParams($fd->getName(), $language, ['isUpdate' => $isUpdate]);
```
Comment (blankse) [inline models/DataObject/Localizedfield/Dao.php:160]: This should stay ;)

```suggestion
                    $childParams = $this->getFieldDefinitionParams($fd->getName(), $language, ['isUpdate' => $isUpdate, 'context' => $context]);
```

Comment (weisswurstkanone) [reply models/DataObject/Localizedfield/Dao.php:160]: travis will complain ;-)

**Pull request merged**